File: sample_player.py
import sys

def fprint(*args):
	print(*args)
	sys.stdout.flush()

fprint('python')

each, width, height, colors = input().split()
width = int(width)
height = int(height)
colors = int(colors)
table = [[None] * height for i in range(width)]


while True:
	cmd = input().strip()
	if cmd == 'exit': exit()
	if cmd == 'play':
		for i in range(width):
			table[i] = list(map(int, input().split()))

		for i in range(colors - 1, -1, -1):
			if i != table[0][0] and i != table[-1][-1]:
				fprint(i)
				break

File: CeilBattler-win32-x64/resources/app/sample_player.cpp
#include <iostream>
#include <vector>

using namespace std;


string each;
int width, height, colors;
vector<vector<int>> field;

// table already reversed
pair<int, int> get_self() {
	return make_pair(0, 0);
}

int main() {
	cout << "Sample Player" << endl;
	cin >> each >> width >> height >> colors;
	string cmd;
	field.resize(width, vector<int>(height));
	while (cin >> cmd) {
		if (cmd == "close") return 0;
		if (cmd == "play") {
			for (int i = 0; i < width; i++) {
				for (int j = 0; j < height; j++) {
					cin >> field[i][j];
				}
			}

			for (int i = 0; i < colors; i++) {
				if ((i != field[0][0])
					&& (i != field[width - 1][height - 1])) {
					cout << i << endl;
					break;
				}
			}
		}
	}
}